diff -up photoflare-1.6.0/PhotoFlare.pro.omv~ photoflare-1.6.0/PhotoFlare.pro
--- photoflare-1.6.0/PhotoFlare.pro.omv~	2019-09-29 17:06:41.343777427 +0200
+++ photoflare-1.6.0/PhotoFlare.pro	2019-09-29 17:07:06.889172347 +0200
@@ -29,7 +29,7 @@ win32 {
 # Project settings for Linux and Hurd. Adjust the paths as needed on your system.
 linux|hurd {
     INCLUDEPATH += /usr/include/GraphicsMagick
-    LIBS += -L/usr/lib -lGraphicsMagick++
+    LIBS += -lGraphicsMagick++
     QMAKE_CXXFLAGS += -fopenmp
     LIBS += -fopenmp
 }
