diff -up photoflare-1.6.1.0/Photoflare.pro photoflare-1.6.1.0/Photoflare.pro.omv
--- photoflare-1.6.1.0/Photoflare.pro   2019-12-22 15:07:38.149332739 +0100
+++ photoflare-1.6.1.0/Photoflare.pro.omv       2019-12-22 15:07:50.408352798 +0100
@@ -29,7 +29,7 @@ win32 {
 # Project settings for Linux and Hurd. Adjust the paths as needed on your system.
 linux|hurd {
     INCLUDEPATH += /usr/include/GraphicsMagick
-    LIBS += -L/usr/lib -lGraphicsMagick++
+    LIBS += -lGraphicsMagick++
     QMAKE_CXXFLAGS += -fopenmp
     LIBS += -fopenmp
 }

